recipes/repeatfinder/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+$CC -std=c99 -g -o repeatfinder main.c conversion.* switches.*
```

Comment:
this command does not work for macos, which uses the `clang` compiler, fails with 
```
INFO:bioconda_utils.utils:(ERR) clang-12: error: cannot specify -o when generating multiple output files
```